website/docs/features2.mdx

```diff
@@ -281,6 +280,13 @@ To investigate the convergence of your multi-objective optimization. See below f
 <img alt="moo distrb plot" src={useBaseUrl('img/moo_distrb_plot.png')} />
 <img alt="moo cloud plot" src={useBaseUrl('img/moo_cloud_plot.png')} />
 
+---
+## Time Series Validation
+
+Time series validation parameter (train_size) is included as one of the four components in Robyn's hyperparameters. When ts_validation = TRUE in robyn_run(), train_size defines the percentage of data used for training, validation and out-of-sample testing. For example, when train_size = 0.7, val_size and test_size will be 0.15 each. This hyperparameter is customizable with default range of c(0.5, 0.8) and must be between c(0.1, 1).
```

Comment:
To format parameters and functions as code, we can wrap them up with ` (backtick quotes)

Reply:
This is an `example`